File: sdk-node/examples/contacts/count.ts
/**
 * Get the total contact count for the host.
 *
 * Run:
 *   npx tsx examples/contacts/count.ts
 */

import { ThreeCommon } from '@3common/sdk'

const client = new ThreeCommon({
  apiKey: '3co_your_api_key_here',
})

const { count } = await client.contacts.count()

console.log(`host has ${String(count)} contacts`)

File: sdk-python/examples/contacts/count.py
"""Run with: python examples/contacts/count.py"""

from __future__ import annotations

from threecommon import ThreeCommon


def main() -> None:
    with ThreeCommon(api_key="3co_your_api_key_here") as client:
        result = client.contacts.count()
        print(f"host has {result.count} contacts")


if __name__ == "__main__":
    main()
